quasar.conf.js

```diff
@@ -35,13 +35,9 @@ module.exports = function (ctx) {
       { path: 'event-buss', server: false }
     ],
 
-    css: [
-      'app.styl'
-    ],
+    css: ['app.styl'],
```

Comment:
Seems like we need to make everyone set the same style guides in their IDEs / editors.